pkg/rotation/reconciler.go

```diff
@@ -286,50 +314,65 @@ func (r *Reconciler) reconcile(ctx context.Context, spcps *v1alpha1.SecretProvid
 		}, updateFn); err != nil {
 			return fmt.Errorf("failed to update spc pod status, err: %+v", err)
```

Comment:
Do we also want to generate an event for this failure too?

Reply:
Yeah, added an event for failing to spcps with the latest versions.